README.md

```diff
@@ -86,13 +92,17 @@ In order to spin-up operating instances of keepers running the code previously d
 yarn gelato:deploy
 ```
 
-This script will check the contents of the `pre-deployments.json` file and the `deployments.json` file and determine for which keeper types to run the actual deployment script. The deployment will not be triggered if there is no IPFS deployment address for a given keeper, or the the IPFS deployment address in the `pre-deployments.json` file matches the one in the `deployments.json` file, which means, that this version of the keeper code was already deployed. Otherwise, the deployment will be performed.
+This script will check the contents of the `pre-deployments.json` file and determine for which keeper types to run the actual deployment script. The deployment will not be triggered if there is no IPFS deployment address for a given keeper, or a given IPFS deployment, combined with a configuration is already deployed. Otherwise, the deployment will be performed.
+
+The exact number of instances, their names, arguments, and configuration are defined in config files. The arguments used to spin-up the keeper are passed along with the deployment transaction and can only be changed by redeploying the keeper instance. Secrets are set in separate transactions, after the initial deployment and their value can be changed at any time via UI or a script.
 
-The exact number of instances, their names, arguments, and configuration are defined in the script. The arguments used to spin-up the keeper are passed along with the deployment transaction and can only be changed by redeploying the keeper instance. Secrets are set in separate transactions, after the initial deployment and their value can be changed at any time via UI or a script.
+Any time, the script is run, all previously running keepers are retired, unless an exactly the same keeper, with exactly the same configuration would be deployed.
```

Comment:
Any time the script is run

Reply:
unless keepers that have the same code logic (IPFS hash) and configuration

Reply:
Any time the script is run, all previously running keepers are retired, unless keepers that have the same code logic (IPFS hash) and configuration.